js/src/data/types.js

```diff
@@ -9,7 +9,33 @@
  * @property {string[]} square_marketing_images The URLs of the square images.
  * @property {string[]} logo The URLs of the logo images.
  * @property {string[]} display_url_path The path part of the display URL on the ad.
- * @property {string | null} call_to_action The call-to-action text on the ad to let users know what the ad will get them to do. `null` if not selected.
+ * @property {string | null} call_to_action_selection The call-to-action text on the ad to let users know what the ad will get them to do. `null` if not selected.
+ */
+
+/**
+ * @typedef {Object} AssetEntity
+ * @property {number} id The ID of the asset.
+ * @property {string} content The content of the asset.
+ */
+
+/**
+ * @typedef {Object} AssetsDictionary
+ * @property {AssetEntity} [business_name] The name of merchant's business or brand.
+ * @property {AssetEntity[]} [headline] The headlines for the ad.
+ * @property {AssetEntity[]} [long_headline] The headlines for the larger ad.
+ * @property {AssetEntity[]} [description] The descriptive text for the ad to provide additional context or details.
+ * @property {AssetEntity[]} [marketing_images] The URLs of the landscape images.
+ * @property {AssetEntity[]} [square_marketing_images] The URLs of the square images.
```

Comment:
These property names are incorrect and have been fixed in PR #1874. Ref: https://github.com/woocommerce/google-listings-and-ads/pull/1873#discussion_r1099983959